shared/chat/conversations-list/container.js

```diff
@@ -68,15 +102,16 @@ class ConversationListContainer extends Component {
   }
 }
 
-function _filterInboxes (inboxes: List<InboxState>, supersededByState: SupersededByState): List<InboxState> {
+function _filterInboxes (inboxes: List<InboxState>, supersededByState: SupersededByState, alwaysShow: Set<ConversationIDKey>): List<InboxState> {
   // $FlowIssue with records and accessing things inside them
-  return inboxes.filter(i => (!i.isEmpty || i.youCreated) && !supersededByState.get(i.conversationIDKey))
+  return inboxes.filter(i => (!i.isEmpty || alwaysShow.has(i.conversationIDKey)) && !supersededByState.get(i.conversationIDKey))
```

Comment:
I don't know what you mean by the 'maybe we should move this into a helper'. The metadata map line?